packages/common/src/interfaces/pageInfo.interface.ts

```diff
@@ -0,0 +1,13 @@
+export interface PageInfo {
```

Comment:
I wonder if we should rename this to `CursorPageInfo` instead since this seems to be related to cursor only?